examples/inspec-linux-audit/test/scripts/provisioner.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+if ! [ -f /tmp/results/hab_artifact.tar.gz ]; then
+  echo "ERROR: No hab_artifact.tar.gz found, did you run 'hab pkg build .'?"
+  exit 1
+fi
+
+export HAB_LICENSE="accept-no-persist"
+export CHEF_LICENSE="accept-no-persist"
+
+# Load variables for use in this script
+source /tmp/results/last_build.env
+
+# Extract Habitat artifact (contains `hab` binary and all keys/packages)
+sudo tar xf /tmp/results/hab_artifact.tar.gz -C /
+
+# Create `hab` user and group if they do not exist
+if grep "^hab:" /etc/passwd > /dev/null; then
+  echo "Hab user exists, skipping user creation"
+else
+  # NOTE: `useradd` also creates a group by default
+  sudo useradd hab && true
+fi
+
+echo "Determine pkg_prefix for $pkg_artifact"
+pkg_prefix=$(find "/hab/pkgs/$pkg_origin/$pkg_name" -maxdepth 2 -mindepth 2 | sort | tail -n 1)
+echo "Found: $pkg_prefix"
+
+# Execute InSpec inside the package (ignoring control failures)
+echo "Running Chef InSpec in $pkg_origin/$pkg_name"
+cd "$pkg_prefix" || exit 1
+
+sudo -E /hab/bin/hab pkg exec "$pkg_origin/$pkg_name" \
+                                inspec exec "$pkg_prefix/*.tar.gz" \
+                                --reporter json:/tmp/inspec_results.json
```

Comment:
This is great.

Reply:
In `examples/inspec-linux-audit/test/integration/default/default_test.rb` does the json only exist if the tests pass?

Reply:
It should be that it should exist if InSpec succeeds to run. But would need to test to be 100% sure.

Reply:
Nope, that file will contain a report with control failures. We only care that InSpec ran and not that the Kitchen VM is secure (spoiler alert: it isn't 😄)